Darkest Crusade/Assets/Dungeon/MainCamera.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MainCamera : MonoBehaviour {
+    public float speed;
+    public float clampLeft;
+    public float clampRight;
+
+    private float cameraX;
+
+    void Start () {
+        cameraX = transform.position.x;
+	  }
+	
+	void Update () {

```

Comment:
This camera control script will not working on phone, please fix it

Reply:
👍